src/app/app.component.html

```diff
@@ -1,12 +1,13 @@
 <ng-container>
     <ngx-loading-bar color="red" [includeSpinner]="false"></ngx-loading-bar>
-    <div class="app-root" [ngClass]="{'logged-in': userLoggedIn}">
-        <div *ngIf="userLoggedIn; then loggedIn else notLoggedIn"></div>
+    <div class="app-root" [ngClass]="{'logged-in': isAppLoadedAndUserLoggedIn()}">
```

Comment:
Do we need a function call here? If I'm right, a function is called continuously and component var is checked only if there is a change in its value? can you verify this?

Reply:
Done